### PR TITLE
Add Django 3.0 support

### PR DESCRIPTION
**Pull request by hugorodgerbrown**
Not expecting this  to be merged, but thought it might be useful. The `<3.0` restriction in `setup.py` is preventing us from testing compatibility of our own project, so I wanted to see how complicated it would be to remove.

The only change required to support the latest version is the replacement of the vendored
`django.utils.six` module with `six` itself.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -39,7 +39,8 @@
                    'Topic :: Utilities'],
       python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
       install_requires=[
-          'Django>=1.11,<3.0',
+          'Django>=1.11,<4.0',
           'requests',
+          'six'
```
Comment (jdufresne) [inline setup.py]: I'm fine either way, but an alternative would be to follow Django's lead and drop Python 2 support as was done in #20. If that is still not desired at this time, this change is very useful for testing future compatibility.

Comment (hugorodgerbrown) [reply setup.py]: I'm happy to drop Py2. Is there any chance of getting this merged (with / without Py2) - as Django 3 is imminent, and this is currently a blocker for it.

**Comment by hugorodgerbrown** [comment]
Updated to Django==3.0

**Comment by ubernostrum** [comment]
I've just pushed commits which test against Django 3.0. Will roll a release from it soon.

**Pull request closed without merging**